### DISCUSSION TITLE
Herramientas para hacer la documentación

### DISCUSSION THREAD

**Discussion by ma-robles** [grupoioa/gis_agm_series#15] [Q&A]
Por el momento se tienen las siguientes propuestas para realizar la documentación:
- Usar google docs (creo que no requiere presentación)
- Markdown. Se integra directamente con github, es de fácil uso, se puede exportar a html y otros formatos (como docx).
- Cuau propone usar [sphinx](https://www.sphinx-doc.org/en/master/)

**Comment by MaCuur** [comment]
esas se las presentamos mañana al doc. Una vez que termine les mando una pequeña presentación =)

**Reply by ma-robles** [reply]
Yo creo que es mejor tomar esa decisión nosotros, a final de cuentas tenemos que generar la documentación en el formato y requirimientos que nos pidan. El "cómo" no creo que sea importante para el doc, siempre y cuando sea de una manera eficiente. Probablemente podemos preguntar los requisitos que va a llevar esa dicumentación. La flecha creo que es para votar :+1:

**Reply by MaCuur** [reply]
ahh va va... muchas gracias ⬆️ 
Entonces solo le preguntamos que cual es el plan.

**Reply by ma-robles** [reply]
mmm pues el plan también es cuestión de nosotros en gran parte, mas bien debemos preguntar los requerimientos y verificar que el plan va de acuerdo con lo que se necesita. jeje :)

**Reply by AbrilIchel** [reply]
En un rato mas les envío la liga donde estén todos los informes de PEMEX editables, para que se basen en ese formato y estructura

**Reply by ma-robles** [reply]
muy bien @AbrilIchel , aunque están bastante simples (me parece)
